packages/docusaurus-plugin-content-blog/src/__tests__/pluginOptionSchema.test.ts

```diff
@@ -82,6 +82,23 @@ test('should convert all feed type to array with other feed type', () => {
   });
 });
 
+test('should contain array with rss + atom for missing feed type', () => {
+  const {value} = PluginOptionSchema.validate({
+    feedOptions: {},
+  });
+  expect(value).toEqual(DEFAULT_OPTIONS);
+});
+
+test('should have array with rss + atom, title for missing feed type', () => {
+  const {value} = PluginOptionSchema.validate({
+    feedOptions: {title: 'title'},
+  });
+  expect(value).toEqual({
+    ...DEFAULT_OPTIONS,
+    feedOptions: {type: ['rss', 'atom'], title: 'title'},
+  });
+});
+
```

Comment:
That looks good

Can you please add a test to ensure that type: null leads to type: null after validation? To ensure it's not using default values as a fallback?